storage/streams/src/main/java/io/apicurio/registry/streams/utils/LongGenerator.java

```diff
@@ -0,0 +1,67 @@
+package io.apicurio.registry.streams.utils;
+
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicLong;
+
+@FunctionalInterface
+public interface LongGenerator {
+
+    long getNext();
+
+    class HiLo implements LongGenerator {
```

Comment:
Is this actually used?

Reply:
No. I used it before, but realised proper usage of <offset, partition> is good enough.
Since I went through the trouble to impl it, just didn't want it to go to waste, in case I do need it later on.

Reply:
It's your repo, so I'm not going to tell you not to keep dead code in it, but you could always just push it to a selarate branch until such time as you've decided whether you really need it.

Reply:
I'll re-check if I'm still better off with using this HiLo ... but yeah, I can move it into a branch, and clean it in master.